src/components/api-request.js

```diff
@@ -175,6 +175,7 @@ export default class ApiRequest extends LitElement {
                   <tag-input class="request-param" 
                     autocomplete="on"
                     id = "request-param-${paramName}"
+                    aria-labelledby = "request-${paramName}"
```

Comment:
We update the `id` so this should be:

```suggestion
                    aria-labelledby = "request-${paramName}-label"
```

Same goes for all the other ones.

Reply:
Updated.